launch_ros/launch_ros/actions/load_composable_nodes.py

```diff
@@ -0,0 +1,202 @@
+# Copyright 2019 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Module for the LoadComposableNodes action."""
+
+import logging
+from typing import List
+from typing import Optional
+
+import composition_interfaces.srv
+
+from launch.action import Action
+from launch.actions import RegisterEventHandler
+from launch.event_handlers.on_process_start import OnProcessStart
+from launch.events.process import ProcessStarted
+from launch.launch_context import LaunchContext
+from launch.some_substitutions_type import SomeSubstitutionsType
+from launch.utilities import normalize_to_list_of_substitutions
+from launch.utilities import perform_substitutions
+
+from .composable_node_container import ComposableNodeContainer
+
+from ..descriptions import ComposableNode
+from ..utilities import evaluate_parameters
+from ..utilities import to_parameters_list
+
+_logger = logging.getLogger(name='launch_ros')
+
+
+class LoadComposableNodes(Action):
```

Comment:
I was expecting this to be singular `LoadComposableNode`. I can't think of a downside to this approach though.

Reply:
`LoadComposableNode` sounds good too.

Reply:
Ah, I meant I expected an action `LoadComposableNode` that loaded only one composable node instead of taking a list of them. No need to change the class name like was done in fffac89. The plural name fits better since it can load multiple composable nodes.

Reply:
Argh, I misunderstood you.

Reply:
Now that I'm re-reading ros2/launch#168 I see the example action is `LoadComposableNodes` plural. I have the memory of a goldfish :man_shrugging: